src/pages/hardware/developingAnIOX/messagingProtocol.tsx

```diff
@@ -0,0 +1,1361 @@
+                </tr>
+            </tbody>
+        </table>
+        <h3>Wakeup (0x04)</h3>
+        <p>Wakes up all the IOXs from Sleep Mode. Is sent by the GO at least twice within a space of 50 ms. Currently the GO device sends this message 5 times with 10 ms intervals.</p>
+        <h3>Sleep (0x05)</h3>
+        <p>Causes all IOXs to go into Sleep Mode. Devices will enter Sleep Mode no sooner than 2 seconds, and not more than 20 seconds, after receiving this command. In the meantime, they will report through the poll response that they are going to sleep.</p>
+        <h3>TX Data (0x0B)</h3>
+        <p>Data sent from the GO device to the addressed IOX. The contents of this payload may follow a higher level protocol structure such as <a href="/sdk/hardware/developing-an-iox/mime-protocol/">MIME</a>.</p>
+        <h4>Payload - TX Data</h4>
+        <table>
+            <thead>
+                <tr>
+                    <th>Byte #</th>
+                    <th>Byte Description</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>0-7</td>
+                    <td>Data to transmit</td>
+                </tr>
+            </tbody>
+        </table>
+        <h3>RX Data (0x0C)</h3>
+        <p>Data sent from an IOX to the GO device. The GO will reply with an ACK. The contents of this payload may follow a higher level protocol structure such as <a href="/sdk/hardware/developing-an-iox/mime-protocol/">MIME</a>. The 0x0C message series start and end with a Information Type 1 - Packet Wrapper <a href="#iox-requeststatus-0x25">0x25 message</a>.</p>
+        <h4>Payload - RX Data</h4>
+        <table>
+            <thead>
+                <tr>
+                    <th>Byte #</th>
+                    <th>Byte Description</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>0-7</td>
+                    <td>Received data</td>
+                </tr>
+            </tbody>
+        </table>
+        <h3>Acknowledge (0x14)</h3>
+        <p>Sent by the GO to indicate that a message is being acknowledged. The ACK to an RX Data message (0x0C) could include 1 byte of data. This data is used for streaming flow control. When the 80% watermark of the receive buffer has been reached, the flow control bit will tell the IOX to hold off sending for 50 ms. The IOX will send the next frame at the end of this period and, depending on the flow control bit of the ACK, it will either continue sending or delay another 50 ms, thus repeating the process. The GO device will clear the flow control bit whenever the buffer is below the 20% watermark. If transferring data as part of a wrapped packet exchange, the streaming watermark can be ignored. The buffers will not overflow so long as the length limit and the modem result are honored. This byte is only sent when needed.</p>
+        <h4>Payload</h4>
+        <table>
```

Comment:
we should have tables embedded in a div with the .table-container class until we come up with the final styling so it can be applied later on